lib/Distribution/Opensuse/AgamaDevel.pm

```diff
@@ -54,7 +55,7 @@ sub get_agama_up_an_running {
 }
 
 sub get_reboot {
-    return Yam::Agama::Pom::RebootTextmodePage->new() if is_s390x() || is_ppc64le();
+    return Yam::Agama::Pom::RebootTextmodePage->new() if is_s390x() || (is_ppc64le() && !is_qemu);
```

Comment:
LGTM.
an alternative more class-conservative could be i.e. add in `lib/Yam/Agama/Pom/RebootTextmodePage.pm` somethings minimal like:
```
sub expect_is_shown {
    my $self = shift;
    my $timeout = 2400;
    return;
}
````

Reply:
I wouldn't like to overengineer it. We simply need POWER9, at the moment it can be only emulated, and these VMs kinda behave as `qemu` that we know.

Reply:
Only to clarify, it'd be no overengineering, but a fix of a missing parent class. Anyway PR is ok already as simple as is.

Reply:
Ok, I see what you mean. Well ... in fact, it would have to be the same function is in RebootPage. And another point, with emulated ppc64le, we are not dealing with a Textmode scenario.

Reply:
Correction for the future: the `expect_is_shown` could neither be a simple _return_, still a loop needed to verify installation completed 🙂.